import java.util.*;

public class MultipleException {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try {
            System.out.print("Enter a number: ");
            int a = sc.nextInt();
            int b = 12 / a;
            int[] c = new int[b];
            c[4] = 10;
            System.out.println("Array Created");
        } 
        catch (ArithmeticException e) { System.out.println("Divide By Zero"); }
        catch (InputMismatchException e) { System.out.println("Invalid Input"); }
        catch (ArrayIndexOutOfBoundsException e) { System.out.println("Array Index Error"); }

        System.out.println("Program Executed Successfully!");
    }
}
